### DISCUSSION TITLE
How to make a request/replace fragment with a radio button?

### DISCUSSION THREAD

**Discussion by gawry** [unpoly/unpoly#233] [Q&A]
I'm wondering how to make that but it's not working well. Perhaps there is a better way. 

`<input type="radio" name="teste" up-choose up-data='{"href": "/clients/", "target":".form-stack"}'>`

`up.$macro('input[up-choose]', (link, data) => {
    var location = data;
    link.click(function(el){
        console.log("click");
        console.log(location);
        up.replace(location["target"], location["href"],
            {
                layer: "modal",
                fallback: "body"}
        )
    })
});`

ps: I'm still on 0.62 but can update if necessary to solve this.

**Comment by triskweline** [comment]
I can't test your code, but you may want to look into the `[up-validate]` attribute and [Updating dependent fields](https://unpoly.com/input-up-validate#updating-dependent-fields).

In general I recommend against prefixing your custom attributes with `up-` since that namespace is reserved for future Unpoly features.
